src/services/DevicesService.js

```diff
@@ -4,8 +4,8 @@ class DevicesService {
     registerDeviceInteractor,
     unregisterDeviceInteractor,
     authDeviceInteractor,
-    listDevicesInteractor,
     updateSchemaInteractor,
+    listDevicesInteractor,
```

Comment:
Please, keep the order of the parameters similar to the definitions order inside the constructor.

Reply:
It's weird, but it was necessary for code to run correctly. Apparently the code was wrong, check [here](https://github.com/CESARBR/knot-fog-connector/blob/f19ec380818bc2f0b07d623c2ef8857f3f8fe784/src/network/MessageHandlerFactory.js#L32) that there was a mismatch in the parameters order.

What intrigues me is that the code should have thrown some error while executing `updateSchema` interactor. By the way, note that doesn't even exist an `listDevice` interactor anymore...

Reply:
That makes sense, @esdrasjnr. The problem here is that there's leftover code the list devices interactor, which is confusing us (please remove it). However, notice when we construct this class we pass `updateSchemaInteractor` as its 5ª parameter and it's correctly attributed locally.